base-module.gradle

```diff
@@ -20,6 +20,7 @@ android {
 
         buildConfigField "String", "TASKY_API_KEY", "\"${properties.getProperty("TASKY_KEY")}\""
         buildConfigField "String", "DATASTORE_AUTH_KEY", "\"${properties.getProperty("AUTH_KEY")}\""
+        buildConfigField "String", "DATASTORE_NAME_KEY", "\"${properties.getProperty("NAME_KEY")}\""
```

Comment:
You are only going to use it on the DataStorePreferenceStorageImpl
So why not just make it a constant there, instead of spreading it across the whole app?

(Same with the rest of the DataStore)

The API Key makes more sense since you want to hide it from being uploaded to GIT